rsfbclient-native/src/params.rs

```diff
@@ -123,9 +137,66 @@ impl ParamBuffer {
         var.sqldata = buffer.as_mut_ptr() as *mut _;
         var.sqllen = buffer.len() as i16;
 
-        ParamBuffer {
+        Ok(ParamBuffer {
             _buffer: buffer.into_boxed_slice(),
             _nullind: nullind,
+        })
+    }
+}
+
+// Convert the binary vec to a blob
+fn binary_to_blob(
+    buffer: Vec<u8>,
+    db_handle: &mut ibase::isc_db_handle,
+    tr_handle: &mut ibase::isc_tr_handle,
+    ibase: &IBase,
+) -> Result<Vec<u8>, FbError> {
+    let mut status = Status::default();
+    let mut handle = 0;
+
+    let mut blob_id = MaybeUninit::<ibase::GDS_QUAD_t>::uninit();
```

Comment:
You can just initialize with 0 instead of using `MaybeUninit`.

```suggestion
    let mut blob_id = ibase::GDS_QUAD_t { gds_quad_high: 0, gds_quad_low: 0 };
```

Reply:
Done